plugins/com.gwtplugins.gdt.eclipse.suite/src/com/google/gdt/eclipse/suite/wizards/WebAppTemplateProjectCreator.java

```
@@ -0,0 +1,435 @@
+
+      // Set all of the natures on the project
+      NatureUtils.addNatures(project[i], projectTemplate.getNatureIds(i));
+
+      // Create the java project
+      createdJavaProjects[i] = JavaCore.create(project[i]);
+
+      // Create a source folder and add it to the raw classpath
+      IResource warFolder = project[i].findMember(WebAppUtilities.DEFAULT_WAR_DIR_NAME);
+      boolean createWarFolders = (warFolder != null);
+      createFolders(project[i], createWarFolders, monitor);
+      if (createWarFolders) {
+        // Set the WAR source/output directory to "/war"
+        WebAppUtilities.setDefaultWarSettings(project[i]);
+
+        // Set the default output directory
+        WebAppUtilities.setOutputLocationToWebInfClasses(createdJavaProjects[i], monitor);
+        /**
+         * Copy files into the web-inf lib folder. This code assumes that it is running in a context that has a workspace
+         * lock.
+         */
+        Sdk gwtSdk = getGWTSdk();
+        setGwtSdk(gwtSdk);
+        if (gwtSdk != null) {
+          new GWTUpdateWebInfFolderCommand(createdJavaProjects[i], gwtSdk).execute();
+        }
+      }
+      reformatJavaFiles(dir);
+
+      // Allow other extensions to run after project creation
+      includeExtensionPartipants();
+    }
+  }
+
+  /**
+   * Replaces all occurrences of the PACKAGENAME with the given package name.
+   * @param iProject
+   * @param dir
+   * @throws IOException
+   */
+  private void replacePackageNames(IProject iProject, File dir) throws IOException {
+    Stack<File> files = new Stack<>();
+    files.add(dir);
+    while(files.isEmpty() == false)
+    {
+      File file = files.pop();
+      if(file.getName().contains("_PACKAGENAME_"))
+      {
+        String name = file.getName();
+        name = name.replace("_PACKAGENAME_", packageBaseName.replace('.', File.separatorChar));
+        File newFile = new File(file.getParentFile(), name);
+        newFile.getParentFile().mkdirs();
```

consider using java.nio.Files.createDirectories(..) here - the Files class will throw instead of silently fail (or letting you ignore the return value, as you're doing here).

---

changed to createDirectories